Sources/Interpreter/Memory.swift

```diff
@@ -278,6 +278,33 @@ public struct Memory {
     throw Error.noDecomposable(t, at: a)
   }
 
+  /// Copies `n` bytes from `source` to `destination`.
+  public mutating func copy(
+    bytes n: Int, from source: Memory.Address, to destination: Memory.Address
```

Comment:
```suggestion
    byteCount n: Int, from source: Memory.Address, to destination: Memory.Address
```

`n` is not the bytes.  However, you could:


```suggestion
    _ n: Int, bytesFrom source: Memory.Address, to destination: Memory.Address
```